submodules/wizard/wizard.js

```diff
@@ -861,24 +878,204 @@ define(function(require) {
 			};
 		},
 
+		/* CREDIT BALANCE AND FEATURES */
+
+		/**
+		 * Render Credit Balance + Features step
+		 * @param  {Object} args
+		 * @param  {Object} args.data  Wizard's data that is shared across steps
+		 * @param  {Object} args.data.usageAndCallRestrictions  Data specific for the current step
+		 * @param  {jQuery} args.container  Step container element
+		 */
 		wizardCreditBalanceAndFeaturesRender: function(args) {
 			var self = this,
-				$container = args.container;
+				$container = args.container,
+				initTemplate = function() {
+					var controlCenterFeatureTree = [
+							{
+								category: 'settings',
+								features: [
+									{
+										name: 'user',
+										icon: 'user'
+									},
+									{
+										name: 'account',
+										icon: 'avatar--badge'
+									}
+								]
+							},
+							{
+								category: 'billing',
+								features: [
+									{
+										name: 'billing',
+										icon: 'credit-card'
+									},
+									{
+										name: 'balance',
+										icon: 'list',
+										features: [
+											{
+												name: 'credit',
+												icon: 'available-balance'
+											},
+											{
+												name: 'minutes',
+												icon: 'clock'
+											}
+										]
+									},
+									{
+										name: 'service_plan',
+										icon: 'service-plan'
+									},
+									{
+										name: 'transactions',
+										icon: 'billing'
+									}
+								]
+							},
+							{
+								category: 'trunking',
+								features: [
+									{
+										name: 'inbound',
+										icon: 'phone-inbound'
+									},
+									{
+										name: 'outbound',
+										icon: 'phone-outbound'
+									},
+									{
+										name: 'twoway',
+										icon: 'two-way'
+									}
+								]
+							},
+							{
+								category: 'misc',
+								features: [
+									{
+										name: 'error_tracker',
+										icon: 'bug'
+									}
+								]
+							}
+						],
+						featureTreeToList = function(features) {
+							return _.flatMap(features, function(feature) {
+								var flattenedFeatures = _.concat([ feature.name ], featureTreeToList(feature.features));
+								return flattenedFeatures;
+							});
+						},
+						controlCenterFeatureList = _
+							.chain(controlCenterFeatureTree)
+							.flatMap(function(category, index) {
+								return _.map(category.features, function(feature) {
+									return _.merge({
+										categoryIndex: index	// TODO: Calculate top position based on category index and feature index (42px for category 0, 50px for the rest, and 40px per feature index)
```

Comment:
What about this TODO?

Reply:
I forgot about it. At the end it was not necessary, so I cleaned up that block.